src/lib.rs

```diff
@@ -85,16 +85,19 @@
 //!
 //! * **serde**: Serialization/Deserialization of entities
 //! * **tracing**: Logging using the `tracing` crate
+//! * **mime03**: Allow creating a [`ContentType`] from an existing [mime 0.3] `Mime` object
```

Comment:
Technically not an _object_, but a _struct_

Reply:
Ha, fair.